src/test/scala/nl/biopet/utils/LazyCheckTest.scala

```diff
@@ -0,0 +1,15 @@
+package nl.biopet.utils
+
+import nl.biopet.test.BiopetTest
+import org.testng.annotations.Test
+
+class LazyCheckTest extends BiopetTest {
+  @Test
+  def test(): Unit = {
+    val lazyCheck = new LazyCheck("bla")
+    lazyCheck.isSet shouldBe false
+    lazyCheck.get shouldBe "bla"
+    lazyCheck() shouldBe "bla"
```

Comment:
After `Bla: the declaration` our hero bla has been living in seclusion. But now our hero is needed once more... In `Bla: the ultimate test` we will see if Bla will be returned!